projects/plugins/jetpack/modules/sharedaddy.php

```diff
@@ -7,7 +7,6 @@
  * First Introduced: 1.1
  * Major Changes In: 1.2
  * Requires Connection: No
- * Requires User Connection: Yes
```

Comment:
Hi there!

It seems that while this is removed, in the same file we still check if we have a connected user in `jetpack_sharedaddy_configuration_url`.
I think this module should require a User Connection. Could you maybe take another look?

Reply:
Aha, I see what you mean. I think it does require a user connection if you want to link to Calypso for the settings, but you don't have to since the settings are also available in wp-admin.

Reply:
The fallback UI in the admin is maintained.

If you're in offline mode, you use the fallback, otherwise you use calypso. This change should be fine

Reply:
We discussed this and it seems that this module doesn't require a User Connection.
The user connection check in `jetpack_sharedaddy_configuration_url` is legit because if there is a connected user we redirect them to calypso, otherwise use the fallback in wp-admin: `wp-admin/options-general.php?page=sharing`